Sources/StreamVideo/Utils/AudioSession/Extensions/AVAudioSession.CategoryOptions+Convenience.swift

```diff
@@ -14,11 +14,15 @@ extension AVAudioSession.CategoryOptions {
         appIsInForeground: Bool
     ) -> AVAudioSession.CategoryOptions {
         [
-            .allowBluetooth,
+            .allowBluetoothHFP,
             .allowBluetoothA2DP
         ]
     }
 
     /// Category options for playback.
     static let playback: AVAudioSession.CategoryOptions = []
+
+    #if !canImport(AVFoundation, _version: 2360.61.4.11)
+    public static let allowBluetoothHFP = Self.allowBluetooth
```

Comment:
I read this about HFP: "if your app is for high-quality audio (music, streaming, etc.), HFP is undesirable because it degrades audio quality."
Will it have impact on audio quality?